### PR TITLE
[FIX] rework plot_surf_atlas example

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none

Changes proposed in this pull request:

- make better use of SurfaceImage object abstraction
- use more meaningful variable names
- add more headings

### CODE REVIEW

**Comment by Remi-Gau** [comment]
[HTML render of the modified example](https://output.circle-artifacts.com/output/job/be4bb2e6-d835-46d9-a43d-0345c5ebf544/artifacts/0/dev/auto_examples/01_plotting/plot_surf_atlas.html#sphx-glr-auto-examples-01-plotting-plot-surf-atlas-py)

File: examples/01_plotting/plot_surf_atlas.py [hunk 1]
```diff
@@ -42,24 +42,21 @@
...
-print(
-    "Fsaverage5 pial surface of left hemisphere is: "
-    f"{fsaverage_meshes['pial'].parts['left']}"
-)
-print(
-    "Fsaverage5 inflated surface of left hemisphere is: "
-    f"{fsaverage_meshes['inflated'].parts['left']}"
-)
+print(f"{fsaverage_meshes['pial'].parts['left']=}")
+print(f"{fsaverage_meshes['inflated'].parts['left']=}")
```
Comment (bthirion) [inline examples/01_plotting/plot_surf_atlas.py:46]: I was not aware of that.

**Review by bthirion — APPROVED** [review]
LGTM, thx.

File: examples/01_plotting/plot_surf_atlas.py (hunk 1 above)
Comment (Remi-Gau) [reply examples/01_plotting/plot_surf_atlas.py:46]: this is a convenient little feature of f-strings: very useful when print debugging

**Pull request merged**